test/OSX.hs

```diff
@@ -4,6 +4,18 @@
 module OSX(sdot) where
 
 import Foreign.Ptr
+import Foreign.Marshal.Alloc
+import Foreign.Storable
 
-foreign import ccall "cblas_sdot"
-   sdot :: Int -> Ptr Float -> Int -> Ptr Float -> Int ->  Float
+foreign import ccall "sdot_"
```

Comment:
@dlewissandy It would be interesting if you could measure the marshaling cost in a future version.